packages/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -41,43 +117,108 @@
 			<span
-				class="hidden text-white font-sans group-hover:inline absolute -top-1 left-[105%] text-sm z-10 px-2 py-1 bg-gray-800/80 leading-relaxed min-w-[150px] w-max max-w-[400px] rounded-md"
+				class="group inline-flex items-center relative cursor-help cursor-helpfont-sans px-1 border border-red-200 py-[1px] bg-red-50 rounded"
 			>
-				{$query.error}
+				<span class="inline font-sans font-medium text-xs text-red-600">error</span>
+				<span
+					class="hidden text-white font-sans group-hover:inline absolute -top-1 left-[105%] text-sm z-10 px-2 py-1 bg-gray-800/80 leading-relaxed min-w-[150px] w-max max-w-[400px] rounded-md"
+				>
+					{$query.error}
+				</span>
 			</span>
-		</span>
-	{:else}
-		<select
-			disabled={hasQuery && !$query.loaded}
-			on:change={(e) => ($inputs[name] = e.currentTarget.value)}
-			class="border border-gray-300 bg-white rounded-lg p-1 mt-2 px-2 pr-5 flex flex-row items-center max-w-fit bg-transparent cursor-pointer bg-right bg-no-repeat"
-			style="background-image: url('data:image/svg+xml,%3Csvg xmlns=\'http://www.w3.org/2000/svg\' class=\'icon icon-tabler icon-tabler-chevron-down\' width=\'18\' height=\'18\' viewBox=\'0 0 24 24\' stroke-width=\'2\' stroke=\'currentColor\' fill=\'none\' stroke-linecap=\'round\' stroke-linejoin=\'round\'%3E%3Cpath stroke=\'none\' d=\'M0 0h24v24H0z\' fill=\'none\'/%3E%3Cpath d=\'M6 9l6 6l6 -6\' /%3E%3C/svg%3E');"
-			value={$inputs[name] || defaultValue}
-		>
-			<slot />
-
-			{#if hasQuery}
-				{#each $query as { label, value }}
-					<DropdownOption {value} valueLabel={label} />
-				{/each}
-			{/if}
-		</select>
-	{/if}
-</div>
+		{:else}
+			<Popover.Root bind:open>
+				<Popover.Trigger asChild let:builder>
+					<Button
+						builders={[builder]}
+						variant="outline"
+						role="combo-box"
+						size="sm"
+						class="min-w-5 h-8 border"
+					>
+						{$selectedValues.length > 0 && !multiple ? $selectedValues[0].label : title}
+						<CaretSort class="ml-2 h-4 w-4" />
+						{#if $selectedValues.length > 0}
+							{#if multiple}
+								<Separator orientation="vertical" class="mx-2 h-4" />
+								<Badge variant="secondary" class="rounded-sm px-1 font-normal sm:hidden">
+									{$selectedValues.length}
+								</Badge>
+								<div class="hidden space-x-1 sm:flex">
+									{#if $selectedValues.length > 3}
+										<Badge variant="secondary" class="rounded-sm px-1 font-normal">
+											{$selectedValues.length} Selected
+										</Badge>
+									{:else}
+										{#each $selectedValues as option}
+											<Badge variant="secondary" class="rounded-sm px-1 font-normal"
+												>{option.label}</Badge
+											>
+										{/each}
+									{/if}
+								</div>
+							{/if}
+						{/if}
+					</Button>
+				</Popover.Trigger>
+				<Popover.Content class="w-[200px] p-0" align="start" side="bottom">
+					<Command.Root shouldFilter={false}>
+						<Command.Input placeholder={title} bind:value={search} />
```

Comment:
```suggestion
       {#if items.length > 5}
						<Command.Input placeholder={title} bind:value={search} />
			 {/if}
```

maybe if the options are short we should hide search input?